lib/TeleIrcException.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Exception that gets thrown when there's an error.
+ */
+class TeleIrcException {
+
+  // ---------------- Constructor ----------------
+
+  /**
+   * Constructor.
+   * @param errorCode - Unique number for this error.
+   * @param message - Corresponding Message.
+   */
+  constructor(errorCode, message) {
+    this.errorCode = errorCode;
+    this.message = message;
+  }
+
```

Comment:
This was probably missing in the first place before the move, but we should put

// ---------------- Functions ----------------

Here to go here so we are consistent with other files.

Reply:
I can add that back -- I ran js-beautify on them

Reply:
Fixed.

Reply:
@xforever1313 also re: the functions thing -- No need... seems silly.